v2_explainer.md

```diff
@@ -0,0 +1,83 @@
+# Picture-in-Picture V2 Explained
+
+## Background: Picture-in-Picture V1
+
+Picture-in-Picture launched for HTMLVideoElement early in 2019. The API was
+designed to be extensible, like the Fullscreen API but at the moment is only
+defined for video elements. The implementation takes the video layer and moves
+it into a window that follows display rules specific to Picture-in-Picture
+(always on top, fixed aspect ratio, etc.).
+
+## What's Missing from V1?
+
+Various partners expressed frustration around the limitations of the API. The
```

Comment:
s/partners/websites/

Let's also not use "frustrations" as it's fairly negative.

Reply:
Fixed.